tests/core/transaction-utils/test_transaction_signature_validation.py

```diff
@@ -15,54 +16,50 @@
 from evm.vm.forks.homestead.transactions import (
     HomesteadTransaction,
 )
+from evm.vm.forks.spurious_dragon.transactions import (
+    SpuriousDragonTransaction,
+)
 
 from evm.utils.transactions import (
     extract_transaction_sender,
     validate_transaction_signature,
 )
 
 
-from evm.vm.forks.spurious_dragon.transactions import (
-    SpuriousDragonTransaction,
-)
-
-from evm.utils.transactions import (
-    is_eip_155_signed_transaction,
-    validate_eip155_transaction_signature,
-)
+@pytest.fixture(params=['Frontier', 'Homestead', 'SpuriousDragon'])
+def transaction_class(request):
+    if request.param == 'Frontier':
+        return FrontierTransaction
+    elif request.param == 'Homestead':
+        return HomesteadTransaction
+    elif request.param == 'SpuriousDragon':
+        return SpuriousDragonTransaction
```

Comment:
Why not use the classes in the params list? `params=[FrontierTransaction, HomesteadTransaction, SpuriousDragonTransaction]`